source/exe/main.cc

```diff
@@ -20,6 +21,12 @@
 #include "ares.h"
 #include "spdlog/spdlog.h"
 
+#if __cplusplus < 201103L ||                                                                       \
```

Comment:
nit: Can we move this duplication into a file called compiler_requirements.h or whatever and include from both places? We might want to have further requirements later.